printctlr/controller.go

```diff
@@ -17,14 +17,17 @@ package printctlr
 import (
 	"fmt"
 
+	"github.com/wpengine/lostromos/crwatcher"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 )
 
 // Controller provides a crwatcher.ResourceController that prints the events out
 // as the are received. It is a basic implementation that can be used for
-// debuging. It also serves as an example for how you could implement your own
+// debugging. It also serves as an example for how you could implement your own
 // crwatcher.ResourceController.
-type Controller struct{}
+type Controller struct {
+	crwatcher.ResourceController
```

Comment:
declaring Controller implements this interface

Reply:
This doesn't declare that this is an implementation of crwatcher.ResourceController.
https://golang.org/doc/faq#implements_interface has more details about how go doesn't have "implements" declarations. In that guide it also provides a way to make sure we implement the interface if we wanted to do that we could but this in and of itself is not something that is standard in Go or that actually does anything.